docs/basics/options.md

```diff
@@ -642,7 +642,8 @@ excludeOlder: "0s",
 When running a search, this option excludes anything that has been searched
 more recently than this long ago. This option is only relevant in
 `search` mode or in `daemon` mode with [`searchCadence`](#searchcadence) turned
-on.
+on. Previous searches that failed due to a timeout or other error will not be
+exluded.
```

Comment:
```suggestion
on.

:::note

Searches that failed on specific trackers (for example - due to timeout or rate-limiting) will not be marked as having been searched, and thus will not be excluded by this setting for those specific trackers on the next run.
:::
```
I'm not sure I'm completely happy with my wording here, however, using a note block would make this stand out a bit more, and allow us to clarify this better. 

Let me know what you think of the language.